kms/kms.rb

```diff
@@ -0,0 +1,421 @@
+# Copyright 2017 Google, Inc
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in write, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Note: Code samples in this file set constants, and
+#       constants cannot be set inside method definitions in Ruby,
```

Comment:
Doesn't read like a sentence.

❓ *Code samples in this file set constants which cannot be set inside method definitions in Ruby.   To allow for this, code snippets in this sample are wrapped in global lambdas.*